pocs/tests/test_observatory.py

```diff
@@ -69,20 +87,44 @@ def test_bad_mount_driver(config):
         Observatory(simulator=simulator, config=conf, ignore_local_config=True)
 
 
-def test_bad_scheduler(config):
+def test_can_observe_no_scheduler(config):
     conf = config.copy()
-    simulator = ['all']
-    conf['scheduler']['type'] = 'foobar'
-    with pytest.raises(error.NotFound):
-        Observatory(simulator=simulator, config=conf, ignore_local_config=True)
+    cameras = create_cameras_from_config(conf)
+
+    obs = Observatory(cameras=cameras,
+                      scheduler=None,
+                      config=conf
+                      )
 
+    assert obs.can_observe is False
 
-def test_bad_scheduler_fields_file(config):
+
+def test_can_observe_no_cameras(config):
```

Comment:
This test and the next test should follow above. That is, we should be testing for the correct log message with minimal examples added. You will want to do that for the cameras since you created the `can_observe` method and the cameras are already part of that. However see not on mount below.